cluster-support-bot.py

```diff
@@ -92,6 +108,15 @@ def _handle_message(payload):
             logger.error(response)
 
 
+def handle_uuid_mention(text):
+    regex = re.compile('.*([a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}).*', re.I)
+    match = regex.match(text)
+    if match:
+        uuid = match.groups()[0]
+        logger.debug('found {}'.format(uuid))
```

Comment:
nit: `found` -> `UUID mention`?  Just to pick something a bit more unique to this function.

Reply:
:+1: